docs/faq/admin.txt

```diff
@@ -94,14 +94,18 @@ presentation by editing the CSS stylesheet and/or associated image files. The
 site is built using semantic HTML and plenty of CSS hooks, so any changes you'd
 like to make should be possible by editing the stylesheet.
 
+.. _admin-browser-support:
+
 What browsers are supported for using the admin?
 ================================================
 
-The admin provides a fully-functional experience to `YUI's A-grade`_ browsers,
-with the notable exception of IE6, which is not supported.
+The admin provides a fully-functional experience to the recent versions of
+modern, web standards compliant browsers. On desktop this means Chrome, Edge,
+Firefox, Opera, Safari, and others.
 
-There *may* be minor stylistic differences between supported browsers—for
-example, some browsers may not support rounded corners. These are considered
-acceptable variations in rendering.
+On mobile and tablet devices, the admin provides a responsive experience for
```

Comment:
Is it not more accurate to just say "On devices with smaller displays"?

Reply:
Meh. I think it's clear enough.

Reply:
IMO the current wording is fine.